qutebrowser/utils/utils.py

```diff
@@ -915,3 +917,32 @@ def yaml_dump(data, f=None):
         return None
     else:
         return yaml_data.decode('utf-8')
+
+
+def pastebin_version():
+    """Pastebins version and logs to messages"""
+
+    app = qutebrowser.utils.objreg.get('app')
+    http_client = httpclient.HTTPClient()
+
+    def _get_paste_title():
+        return "qute version info {}".format(qutebrowser.__version__)
+
+    @pyqtSlot(str)
+    def _on_paste_version_success(url):
+        qutebrowser.utils.message.info("Version info pastebinned"
+                                       " to: {}".format(url))
```

Comment:
This seems quite cumbersome to get the URL from. Ideally it'd get shown on the page, but since we can't do that easily, I'd say it should be copied to the clipboard (with the button text updated accordingly)?